### PR TITLE
WEBUI-362: fix tabs height

### PR DESCRIPTION
**Pull request by semisse**

### CODE REVIEW

File: elements/nuxeo-browser.html [hunk 1]
```diff
@@ -40,12 +40,16 @@
 
       paper-tabs {
         --paper-tabs-selection-bar-color: transparent;
-        height: auto;
+        max-height: 42px;
```
Comment (Gabez0r) [inline elements/nuxeo-browser.html:43]: Does it need to be a `max-height` instead of a `height`?

**Review by Gabez0r — COMMENTED** [review]
looking good

File: elements/nuxeo-browser.html (hunk 1 above)
Comment (semisse) [reply elements/nuxeo-browser.html:43]: I was trying to avoid having a hardcoded height here, but it won't make a huge difference in this case.

Comment (Gabez0r) [reply elements/nuxeo-browser.html]: I'm ok with either way, was just trying to understand if there was any reason in particular. For example, if there are no tabs being contributed, what happens with one or the other?

**Review by mnixo — APPROVED** [review]

File: elements/nuxeo-browser.html (hunk 1 above)
Comment (semisse) [reply elements/nuxeo-browser.html]: In that edge case, we should go with max-height and make sure height is set to auto so the tab bar won't be visible. Just updated the PR to reflect this.

**Review by mnixo — DISMISSED** [review]

**Review by richardsd — DISMISSED** [review]

**Review by Gabez0r — APPROVED** [review]

**Comment by semisse** [comment]
I suggest we go with min-height for tabs to ensure they have space to increase their height if needed. Since having no tabs is probably an edge case, let's keep it as it is. If needed, later we can always check if the `paper-tabs` have children being contributed and make it display none in the UI.

**Review by mnixo — APPROVED** [review]

**Review by richardsd — APPROVED** [review]

**Pull request merged**